### ISSUE TITLE
Chunkloader [1.20] invalid player data issue

### ISSUE THREAD

**Issue by lays24mc** [SuperMartijn642/ChunkLoaders#89]
**Version Info**
- Minecraft 1.20
- Chunk Loaders 1.2.3-forge-mc-1.20

**Are you using OptiFine?:** 

No

**Description of the Bug**

When the mod is installed, you cannot join the world. You get the error "invalid player data".

**Steps to Reproduce**

Install the mod, create a world and try to join it. 

**Log**
```
[09Juni2023 09:32:54.500] [Netty Local Client IO #0/INFO] [net.minecraftforge.network.NetworkHooks/]: Connected to a modded server.
[09Juni2023 09:32:54.733] [Server thread/ERROR] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: Exception caught during firing event: Cannot invoke "com.supermartijn642.chunkloaders.capability.ChunkLoadingCapability.castServer()" because "capability" is null
	Index: 6
	Listeners:
		0: HIGH
		1: ASM: class dev.architectury.event.forge.EventHandlerImplCommon event(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		2: NORMAL
		3: net.minecraftforge.eventbus.EventBus$$Lambda$4637/0x00000008017ca768@37905efb
		4: net.minecraftforge.eventbus.EventBus$$Lambda$4637/0x00000008017ca768@7c4ce512
		5: ASM: class com.supermartijn642.chunkloaders.capability.PlayerActivityTracker onPlayerJoin(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		6: ASM: class com.supermartijn642.chunkloaders.capability.ChunkLoadingEventHandler onPlayerJoin(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		7: ASM: net.mehvahdjukaar.moonlight.forge.MoonlightForge@3d7cd16c onPlayerLoggedIn(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		8: ASM: net.minecraftforge.common.ForgeInternalHandler@26628731 playerLogin(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		9: ASM: class dev.architectury.networking.forge.NetworkManagerImpl loggedIn(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
		10: ASM: class com.supermartijn642.movingelevators.elevator.ElevatorGroupCapability onJoin(Lnet/minecraftforge/event/entity/player/PlayerEvent$PlayerLoggedInEvent;)V
java.lang.NullPointerException: Cannot invoke "com.supermartijn642.chunkloaders.capability.ChunkLoadingCapability.castServer()" because "capability" is null
	at TRANSFORMER/chunkloaders@1.2.3/com.supermartijn642.chunkloaders.capability.ChunkLoadingEventHandler.onPlayerJoin(ChunkLoadingEventHandler.java:31)
	at TRANSFORMER/chunkloaders@1.2.3/com.supermartijn642.chunkloaders.capability.__ChunkLoadingEventHandler_onPlayerJoin_PlayerLoggedInEvent.invoke(.dynamic)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:315)
	at MC-BOOTSTRAP/net.minecraftforge.eventbus/net.minecraftforge.eventbus.EventBus.post(EventBus.java:296)
	at TRANSFORMER/forge@46.0.2/net.minecraftforge.event.ForgeEventFactory.firePlayerLoggedIn(ForgeEventFactory.java:799)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.players.PlayerList.m_11261_(PlayerList.java:261)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.network.ServerLoginPacketListenerImpl.m_143699_(ServerLoginPacketListenerImpl.java:139)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.network.ServerLoginPacketListenerImpl.m_10055_(ServerLoginPacketListenerImpl.java:126)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.network.ServerLoginPacketListenerImpl.m_9933_(ServerLoginPacketListenerImpl.java:70)
	at TRANSFORMER/minecraft@1.20/net.minecraft.network.Connection.m_129483_(Connection.java:255)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.network.ServerConnectionListener.m_9721_(ServerConnectionListener.java:142)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.MinecraftServer.m_5703_(MinecraftServer.java:909)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.MinecraftServer.m_5705_(MinecraftServer.java:816)
	at TRANSFORMER/minecraft@1.20/net.minecraft.client.server.IntegratedServer.m_5705_(IntegratedServer.java:89)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:663)
	at TRANSFORMER/minecraft@1.20/net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251)
	at java.base/java.lang.Thread.run(Unknown Source)

[09Juni2023 09:32:54.736] [Render thread/INFO] [ChunkBuilder/]: Started 10 worker threads
```

**Comment by SuperMartijn642** [comment]
This is a Forge issue https://github.com/MinecraftForge/MinecraftForge/issues/9526.
A pull request to fix it has already been submitted here https://github.com/MinecraftForge/MinecraftForge/pull/9531. Just have to wait until that gets merged now.

**Comment by lays24mc** [comment]
Thank you for the information. Then I'll wait :)

**Comment by SuperMartijn642** [comment]
Fixed as of Forge 46.0.7.
Thank you for reporting the issue!
